test/format/CONFIG.disagg

```diff
@@ -18,8 +18,6 @@ ops.compaction=0
 ops.salvage=0
 ops.throttle=0
 ops.truncate=0
-# FIXME-WT-14931 should allow prepared operations for layered tables.
-ops.prepare=0
```

Comment:
We should not do this yet, this creates failures in Disagg mode. We should link [WT-15051](https://jira.mongodb.org/browse/WT-15051) and fix this first. As per Hari prepare tranactions are low prior for disagg, we can push this for some time.

Reply:
oh, ok. I have it disabled somewhere else. I'll revert this.